src/sql/workbench/services/query/common/queryRunner.ts

```diff
@@ -276,6 +280,17 @@ export default class QueryRunner extends Disposable {
 		this._onMessage.fire([message]);
 	}
 
+	/**
+	 * Handle a QueryComplete from the service layer
```

Comment:
We're going to be doing this more than just for querycomplete, right?

I'd just remove any indication of when it should be called (the runner doesn't care about that) and instead say what it does when it is called.